cookbooks/travis_ci_connie/metadata.rb

```diff
@@ -8,7 +8,6 @@
 source_url 'https://github.com/travis-ci/packer-templates'
 issues_url 'https://github.com/travis-ci/packer-templates/issues'
 
-depends 'nodejs'
```

Comment:
Connie id now NodeJS free?  Is it because NodeJS is available on all the other image stacks?